### DISCUSSION TITLE
predicting another dataset

### DISCUSSION THREAD

**Discussion by Itamargi1** [MIC-DKFZ/nnUNet#2372] [Q&A]
Hello,

maybe it's a novice question but I did not find info about it - I am training dataset1 and I want the model to make prediction on dataset2. How can I do it? I am using the following command:
`nnUNetv2_predict -d Dataset1 -i /path/to/test_set -o /path/to/predictions -f 0 1 2 3 4 -tr nnUNetTrainer -c 2d -p nnUNetPlans`
there I enter a path to the test set but how does the model know which training set was used for training?

please enlighten me
Thanks all in advance
